### ISSUE TITLE
Error message told me to submit my error here. Probably a common one. 

### ISSUE THREAD

**Issue by dannyhaase** [SubtitleEdit/subtitleedit#2134]
I'm new to SubtitleEdit but really liking what I see so far. Hopefully saves me from buying titlebee, which I got used to. I read that Synchronization > Visual Sync is the best option, and I get this error message when I try. Any ideas? Thanks! 

![subtitle edit error](https://cloud.githubusercontent.com/assets/24425760/20954837/3dc4097e-bc04-11e6-9925-8c87fc5a6076.png)

**Comment by dannyhaase** [comment]
Oh, after I abort this longer error shows. 

![subtitleedit error 2](https://cloud.githubusercontent.com/assets/24425760/20954930/d32344a8-bc04-11e6-82a0-cf0e77e1c580.png)

**Comment by niksedk** [comment]
Sorry, I cannot re-create this... can anybody help?

**Comment by dannyhaase** [comment]
Thanks for trying! I appreciate it!

**Comment by ivandrofly** [comment]
What were you doing exactly? @dannyhaase

**Comment by dannyhaase** [comment]
The only subtitles I could find on subscene.com for The Mindy Project Season 2 were not in sync with my files (don't remember if they're mkv or what, away from home now and can check later if needed).  I inserted the video and the srt subtitles into SubtitleEdit and I followed what I read to use Synchronization > Visual Sync and after I made what I believe is the last click, the error message in the OP opens.

I click sync like it says here, but after I press it I get the error message instead of the action described here:

"Press Sync and let Subtitle Edit do the work of shifting forward, backward, squeezing or expanding the subtitles to match the video. Press Save and you are done!🙂"

And this does not just happen for those specific files. I tried to Visual Sync other files I had and I got the same error.

Let me know if I answered your question or if there's any more info you need.

**Comment by dannyhaase** [comment]
Wanted to clarify that when I said mkv not sure if my files are mkv  or what, I mean the video files, of course.

And my SRT subtitle files are titled exactly the same as the video files, if that matters. It works on Plex that way so I copy video file name into srt file name.

**Comment by ivandrofly** [comment]
@dannyhaase  do you remember which season of **The Mindy Project Season 2** that sub belong to?
I found several subtitle on subscene for **The Mindy Project Season 2** (english).

**Comment by dannyhaase** [comment]
I thought it might be best to record exactly what I do so you can see the exact results. https://youtu.be/mc3AAN50VFU

I do not have the original file names of The Mindy Project video or subtitles files since I used FileBot to match the names automatically, and I got them a long time ago so I don't know how to get the file names from my browser. But since this error occurs regardless of the files I use, I recorded another file.

Here the file name of the video file is: Chicago.Fire.S01E01.720p.BluRay.X264-REWARD 

It's a .mkv

I of course changed the srt subtitle file name so it fits in SubtitleEdit, but the original name of the srt file was Chicago.Fire.S02E01.HDTV.x264-LOL - A Problem House.

It's a .srt

**Comment by dannyhaase** [comment]
I have no idea if this would matter, but I wasn't sure if I've tried other file formats, so I tried an avi file and the error also appears.

**Comment by dannyhaase** [comment]
oh crap, I might have used a s01e01 video on Chicago Fire and s02e01 subtitle, but in my next avi check they are the correct ones, so the problem seems consistent. let me know if another recording or anything else would help. Really appreciate this help you all!

**Comment by dannyhaase** [comment]
haha I went ahead and made a recording of me using the proper files cause I feel embarrassed about accidentally using incorrect ones. :)

https://youtu.be/N4JLj-Hehk8

**Comment by dannyhaase** [comment]
ok, I think I realized my error is that I was stupidly using it wrong. I misread the directions when they said to do the same thing to the second video, the right side, and my scrolling back to the start it obviously won't work. Sorry if anyone wasted their time on this. I appreciate the help, and can blame it on being new to everything here. :) ty.

**Comment by niksedk** [comment]
Still I don't know what caused the error. Perhaps you can install latest version of .net framework: https://www.microsoft.com/en-us/download/details.aspx?id=53344

**Comment by ivandrofly** [comment]
yeah that may be the solution, I've also tried to but no success so far... It looks like the exception is been thrown inside string.Format to TimeCode/ToString()...

**Comment by dannyhaase** [comment]
What caused the error is nothing to worry about. It's just that I'm embarrassingly bad at this. The directions to use visual sync that I read said this:

"5. Now, play the video in a split a bit until someone speaks something. Pause it. Pick the subtitle from the dropdown below that matches what was spoken. **Do the same with the second video player.**"
https://codeyarns.com/2013/02/23/how-to-sync-subtitles-using-subtitle-edit/

So since it said "do the same" and I didn't know anything about how to do this, I just did the same exact thing with the right hand side. So instead of selecting the subtitle that ends the show on the right side, I selected the subtitle again that starts the show on the right side. My skipping back the subtitle all the way to the start in both places causes the error.
